### PR TITLE
Fix #1975: Action Bar Text Size Variation

### LINKED ISSUE #1975: Action Bar Text Size Variation
**Describe the bug**
The size of the text varies for two different screens (until now discovered this problem in the FAQ section). It might be a mistake in the layout

**To Reproduce**
Steps to reproduce the behaviour:
1. Open the Drawer and go to the Help Tab.
2. Click on Frequently Asked Questions, a list of questions will appear.
3. Open any Question while observing the Action Bar.
4. We will see that text size changes abruptly.

**Expected behaviour**
Text size must not vary so that it doesn't look like a glitch

**Screenshots**
Preview of Issue

ezgif-3-a9d4c0492a72

**Device**
- Device/emulator being used : Real Device (Real me 3)
- SDK version : Android 10

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation

Fixes #1975 : Varied text sizes in action bar titles in app screens.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Luffy18346** [comment]
@rt4914 
In this PR, I have remove hard-coded text sizes in the toolbar declaration and replaced them with a style so that whenever we want to change the size (if we ever) then we only have to change in the style rather than doing it individually.

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml [hunk 1]
```diff
@@ -30,7 +30,6 @@
         android:background="?attr/colorPrimary"
         android:fontFamily="sans-serif"
         android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: This toolbar is missing ` app:titleTextAppearance="@style/ToolbarTextAppearance"`

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by FareesHussain — APPROVED** [review]
Other than what @rt4914 mentioned, LGTM

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@Luffy18346 The following files also have hard-coded textsizes in the  tool bar please check
 - [revision_card_activity](https://github.com/oppia/oppia-android/blob/6fba4574392d6108f07e6d3bced40d6183a17ec9/app/src/main/res/layout/revision_card_activity.xml#L48)
 - [topic_fragment](https://github.com/oppia/oppia-android/blob/6fba4574392d6108f07e6d3bced40d6183a17ec9/app/src/main/res/layout/topic_fragment.xml#L53)

**Comment by Luffy18346** [comment]
> @Luffy18346 The following files also have hard-coded textsizes in the tool bar please check
> 
> * [revision_card_activity](https://github.com/oppia/oppia-android/blob/6fba4574392d6108f07e6d3bced40d6183a17ec9/app/src/main/res/layout/revision_card_activity.xml#L48)
> * [topic_fragment](https://github.com/oppia/oppia-android/blob/6fba4574392d6108f07e6d3bced40d6183a17ec9/app/src/main/res/layout/topic_fragment.xml#L53)

@FareesHussain I have talked with @rt4914 about the files that you have mentioned. In above files, `TextView` is used to display title rather than using toolbar to display the title and there are many files in code similar to above 2 files, where custom `TextView ` is used to display the title. Since toolbar is not used, then according to my thinking, custom TextView style might get changed in any case so I have kept hardcoded size in those `TextView `as it is.

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml (hunk 1 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: done.

**Comment by rt4914** [comment]
@Luffy18346 Please make sure that the **Assignee** section is always up-to-date.

**Comment by Arjupta** [comment]
@Luffy18346 you must always mention on an Issue that you want to work on it so that we can assign it to you. This practice prevents two or more contributors to work on the same issue. Also, can you mention a description of changes in the PR description like in this [comment](https://github.com/oppia/oppia-android/pull/2296#issuecomment-749111136). It will be nice if you also attached a Gif or Video file confirming the result of changes

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml (hunk 1 above)
Comment (Arjupta) [reply app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: Have you committed these changes yet?

**Comment by rt4914** [comment]
> @Luffy18346 you must always mention on an Issue that you want to work on it so that we can assign it to you. This practice prevents two or more contributors to work on the same issue. Also, can you mention a description of changes in the PR description like in this [comment](https://github.com/oppia/oppia-android/pull/2296#issuecomment-749111136). It will be nice if you also attached a Gif or Video file confirming the result of changes

@ARJUPTA Nice catch @ARJUPTA

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: @ARJUPTA AND @Luffy18346 No the changes are not yet committed.

**Comment by Luffy18346** [comment]
> @Luffy18346 you must always mention on an Issue that you want to work on it so that we can assign it to you. This practice prevents two or more contributors to work on the same issue. Also, can you mention a description of changes in the PR description like in this [comment](https://github.com/oppia/oppia-android/pull/2296#issuecomment-749111136). It will be nice if you also attached a Gif or Video file confirming the result of changes

Ok I will keep this in mind for next time.

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml (hunk 1 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: @rt4914 and @ARJUPTA sorry it was my mistake. I had committed these changes in another branch. Now, I have committed in this branch.

File: app/src/main/res/layout-sw600dp-land/topic_fragment.xml [hunk 2]
```diff
@@ -50,6 +50,7 @@
           android:singleLine="true"
           android:text="@{String.format(@string/topic_name, viewModel.topicNameLiveData)}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/topic_fragment.xml]: Remove this blank line

File: app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-sw600dp-port/completed_story_list_fragment.xml:33]: Thanks.

File: app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml [hunk 3]
```diff
@@ -30,10 +30,11 @@
         android:background="?attr/colorPrimary"
         android:fontFamily="sans-serif"
         android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
         app:navigationContentDescription="@string/go_to_previous_page"
         app:navigationIcon="?attr/homeAsUpIndicator"
+        app:titleTextAppearance="@style/ToolbarTextAppearance"
         app:title="@string/topics_in_progress"
+        app:titleTextAppearance="@style/ToolbarTextAppearance"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml:36]: It is repeated. Remove this.

File: app/src/main/res/layout-sw600dp-port/topic_fragment.xml [hunk 4]
```diff
@@ -50,6 +50,7 @@
           android:singleLine="true"
           android:text="@{String.format(@string/topic_name, viewModel.topicNameLiveData)}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/topic_fragment.xml]: Remove this blank line

File: app/src/main/res/layout-sw600dp/story_fragment.xml [hunk 5]
```diff
@@ -50,6 +50,7 @@
           android:singleLine="true"
           android:text="@{viewModel.storyNameLiveData}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp/story_fragment.xml]: Remove this blank line

File: app/src/main/res/layout/exploration_activity.xml [hunk 6]
```diff
@@ -54,6 +54,7 @@
             android:requiresFadingEdge="horizontal"
             android:scrollHorizontally="true"
             android:singleLine="true"
+
```
Comment (rt4914) [inline app/src/main/res/layout/exploration_activity.xml]: Remove this blank line

File: app/src/main/res/layout/story_fragment.xml [hunk 7]
```diff
@@ -50,6 +50,7 @@
           android:singleLine="true"
           android:text="@{viewModel.storyNameLiveData}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout/story_fragment.xml]: Remove this blank line

File: app/src/main/res/layout/revision_card_activity.xml [hunk 8]
```diff
@@ -45,6 +45,7 @@
           android:singleLine="true"
           android:textAlignment="viewStart"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout/revision_card_activity.xml]: Remove this blank line

File: app/src/main/res/layout/topic_fragment.xml [hunk 9]
```diff
@@ -50,6 +50,7 @@
           android:singleLine="true"
           android:text="@{String.format(@string/topic_name, viewModel.topicNameLiveData)}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout/topic_fragment.xml]: Remove this blank line

File: app/src/main/res/layout-land/story_fragment.xml [hunk 10]
```diff
@@ -51,6 +51,7 @@
           android:singleLine="true"
           android:text="@{viewModel.storyNameLiveData}"
           android:textColor="@color/white"
+
```
Comment (rt4914) [inline app/src/main/res/layout-land/story_fragment.xml]: Remove this blank line

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Luffy18346 Suggested changes. Also please always run the app and make sure that its working correctly before sending it for review.

**Comment by rt4914** [comment]
@Luffy18346 Update your branch with latest develop too.

File: app/src/main/res/layout-land/story_fragment.xml (hunk 10 above)
Comment (Luffy18346) [reply app/src/main/res/layout-land/story_fragment.xml]: Done

File: app/src/main/res/layout/topic_fragment.xml (hunk 9 above)
Comment (Luffy18346) [reply app/src/main/res/layout/topic_fragment.xml]: Done

File: app/src/main/res/layout/revision_card_activity.xml (hunk 8 above)
Comment (Luffy18346) [reply app/src/main/res/layout/revision_card_activity.xml]: Done

File: app/src/main/res/layout/story_fragment.xml (hunk 7 above)
Comment (Luffy18346) [reply app/src/main/res/layout/story_fragment.xml]: Done

File: app/src/main/res/layout/exploration_activity.xml (hunk 6 above)
Comment (Luffy18346) [reply app/src/main/res/layout/exploration_activity.xml]: Done

File: app/src/main/res/layout-sw600dp/story_fragment.xml (hunk 5 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp/story_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-port/topic_fragment.xml (hunk 4 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp-port/topic_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml (hunk 3 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml:36]: Done

File: app/src/main/res/layout-sw600dp-land/topic_fragment.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout-sw600dp-land/topic_fragment.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**